tests/ui/components/results/paginationcomponent.js

```diff
@@ -144,3 +147,48 @@ describe('rendering the page numbers', () => {
     expect(paginate).toHaveBeenCalledWith(2, 1, 5);
   });
 });
+
+describe('properly interacts with storage', () => {
```

Comment:
Since we're not using an actual instance of `PersistentStorage` or `GlobalStorage`, I think these would qualify as unit tests. As we discussed last night, though, that's fine! We should just make that clear in the commit message.